prover/witness_generator/src/main.rs

```diff
@@ -225,11 +229,8 @@ async fn main() -> anyhow::Result<()> {
             round,
             started_at.elapsed()
         );
-        metrics::gauge!(
-            "server.init.latency",
-            started_at.elapsed(),
-            "stage" => format!("fri_witness_generator_{:?}", round)
-        );
+        let label: String = format!("fri_witness_generator_{:?}", opt.round);
```

Comment:
- Why is it changed to `opt.round` here? AFAIU, it's not always equal to `round`.
- Ideally, we'd want to use a wrapper similar to `StageLabel` here. (Or maybe, `StageLabel` itself if it doesn't cost too much to change dashboards etc.)